Yubico.YubiKey/src/Yubico/YubiKey/IYubiKeyDevice.cs

```diff
@@ -693,5 +693,15 @@ public interface IYubiKeyDevice : IYubiKeyDeviceInfo, IEquatable<IYubiKeyDevice>
         /// </remarks>
         /// <param name="value">The touch threshold to apply to the YubiKey. Must be a value greater than `6`.</param>
         void SetTemporaryTouchThreshold(int value);
+
+        /// <summary>
+        /// Perform device wide reset.
+        /// </summary>
+        /// <remarks>
+        /// <para>
+        /// Only available on YubiKey Bio Multi-protocol.
+        /// </para>
+        /// </remarks>
+        void DeviceReset();
```

Comment:
```suggestion
        void ResetDevice();
```

We like to name things so that they make sense semantically, if we can use imperative form here it adds to the semantics

Reply:
Coming back to this, I agree that it makes sense to keep the same names as the other SDKs